Selectively hide page elements on print

lizgene opened the pull request:
## Why?

Since we use `window.print()` to print the page from the browser right now, it will pull in navigation elements, etc. We don't want these in our final study guide document, so we want to selectively hide them just for print. 

## What?

- Adds class `.no-print` to hide elements within `@media print`
- Adds `.no-print` to current page navigation and the anchor tag to print
- Also adds minor margin changes to the footer, so it's less crunched.

## How can/should this be manually tested?

[Visit any post](http://127.0.0.1:4000/sf-feminist-book-club/sister-outsider/) and click print at the bottom of the page.

## Screenshots (if appropriate)

Before:
![image](https://cloud.githubusercontent.com/assets/4473327/25033887/ce46c760-2096-11e7-84ac-8d5e9e20f221.png)

After: 
![image](https://cloud.githubusercontent.com/assets/4473327/25033869/97112aec-2096-11e7-9c95-943afb6ce8f8.png)

![bitmoji](https://render.bitstrips.com/v2/cpanel/9941680-239426165_2-s1-v1.png?transparent=1&palette=1&width=246)

Commit:
Selectively hide page elements on print

- Adds class no-print to hide elements within @media print
- Adds no-print to current page navigation and the anchor tag to print

pamo commented on _layouts/posts.html:
```diff
@@ -14,9 +14,9 @@
   {{ content }}
-  <footer>
+  <footer class="no-print">
     <a href="javascript:window.print();">Download this post</a>
```
Should we change this link to read `Print this post`?

pamo commented on _sass/_layout.scss:
```diff
@@ -14,5 +14,7 @@ time {
 footer {
   border-top: 1px solid $divider-color;
-  padding: $base-spacing 0;
+  text-align: center;
```
Maybe let's keep this consistent with the back link?

lizgene replied:
Yeah that's reasonable. Removed 👍

lizgene replied:
![image](https://cloud.githubusercontent.com/assets/4473327/25034023/330df2da-2098-11e7-80dd-f446554218a7.png)

Commit:
Update styling of print anchor

- Change copy to 'Print this post'
- Update to left alignment in consistency with top nav alignment

pamo commented on _sass/_layout.scss:
```diff
@@ -15,4 +15,5 @@ time {
 footer {
   border-top: 1px solid $divider-color;
   padding: $base-spacing 0;
+  margin-top: $base-spacing;
```
Shouldn't need this

Commit:
Remove unnecessary top margin on footer
